<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="showQuickViewButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ShowQuickView</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hideQuickViewButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>HideQuickView</string>
   </property>
  </widget>
  <widget class="QWidget" name="stackedLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>221</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QStackedLayout" name="stackedLayout"/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Widget placed underneath QML view</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
